### PR TITLE
Init

### PR DESCRIPTION
**Pull request by Theodus**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2018, The Pony Developers
```
Comment (SeanTAllen) [inline LICENSE]: Copyright is wrong, should just be Pony Developers 2020.

I missed that when copying over from changelog-tool and tweaking.

File: README.md [hunk 2]
```diff
@@ -1 +1,20 @@
-# rfc-tool
\ No newline at end of file
+# rfc-tool
```
Comment (SeanTAllen) [inline README.md:1]: This all needs updating

**Review by SeanTAllen — COMMENTED** [review]
Is there any way to run the tests in an automated fashion?

File: LICENSE (hunk 1 above)
Comment (Theodus) [reply LICENSE]: already fixed in 248fd5e

**Comment by SeanTAllen** [comment]
Thoughts on dropping macOS support @Theodus? If this is all done via GitHub Actions, I doubt we need it.

**Comment by Theodus** [comment]
That sounds fine. There is nothing OS-specific in this repo

**Comment by Theodus** [comment]
> Is there any way to run the tests in an automated fashion?

All the tests are run in `make test`. Is there something else you had in mind?

**Comment by SeanTAllen** [comment]
I didn't see the actual test running code. I guess I missed it?

**Comment by SeanTAllen** [comment]
I'll remove the macOS stuff.

**Comment by SeanTAllen** [comment]
Ah I see how the test works

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -18,22 +18,6 @@ jobs:
...
-        run: brew install ponyc
-      - name: brew install dependencies
-        run: brew install coreutils python
-      - name: pip install dependencies
-        run: pip3 install --upgrade cloudsmith-cli
-      - name: Build and upload
-        run: bash .ci-scripts/release/x86-64-apple-darwin-release.bash
-        env:
-          CLOUDSMITH_API_KEY: ${{ secrets.CLOUDSMITH_API_KEY }}
-
```
Comment (Theodus) [inline .github/workflows/release.yml]: The `x86-64-apple-darwin-release` needs to be removed below in `needs: [x86-64-unknown-linux-release, x86-64-apple-darwin-release, build-release-docker-images]`

Comment (SeanTAllen) [reply .github/workflows/release.yml]: updated

File: README.md (hunk 2 above)
Comment (Theodus) [reply README.md:1]: updated in 9c2e0ad

**Comment by SeanTAllen** [comment]
@Theodus you feeling good about this? shall I squash and merge so we can some nightly dockerfiles and binaries to try out?

i'll let you know if it seems like it would be good to add to ponyup even if we only have a single consumer.

**Comment by Theodus** [comment]
I think this is good to merge, yes.

**Pull request merged**